types/Batch/BatchShipment.d.ts

```diff
@@ -0,0 +1,26 @@
+import { TBatchStatus } from './BatchStatus';
+
+/**
+ * @see https://www.easypost.com/docs/api/node#batch-shipment-object
+ */
+export declare interface IBatchShipment {
+  /**
+   * The id of the Shipment. Unique, begins with "shp_"
+   */
+  id: string;
```

Comment:
I don't think this would be valid. From those docs:

> When used with concrete literal types, a template literal produces a new string literal type by concatenating the contents.

You are trying to concatenate a literal with a `string` type, which is not a literal.

Reply:
I think it should work. It seems to in the playground
![Screen Shot 2022-06-29 at 11 19 55 AM](https://user-images.githubusercontent.com/31320460/176486658-8ec36a11-c476-40fb-980b-ff9a8e90365b.png)

Reply:
But this is not necessary, and more of a future improvement if we wanted to enforce the correct ids being passed into these functions.

Reply:
Ah very cool, I'll have to add that to my toolbelt. Thanks!